lib/teams/server/message-actions/chat-message-actions.js

```diff
@@ -0,0 +1,235 @@
+const { sc, Logger } = require('@reldens/utils');
+const { ChatConst } = require('../../../chat/constants');
+const { TeamsConst } = require('../../constants');
+
+class ChatMessageActions
+{
+    constructor(props)
+    {
+        this.events = sc.get(props, 'events', false);
+        this.chatPlugin = sc.get(props, 'chatPlugin', false);
+        this.client = false;
+        if (!this.events) {
+            Logger.error('EventsManager undefined in Teams ChatMessageActions');
```

Comment:
none return? just log the error? we should have a return for this matter on whatever the class is constructed, to avoid the execution if it has missin properties